app/controllers/warehouse_reports/dv_victim_service_controller.rb

```diff
@@ -51,9 +32,27 @@ def clients
     def client_scope
       GrdaWarehouse::Hud::Client.
         joins(:health_and_dvs, enrollments: :project).
-        where(hdv_t[:InformationDate].gteq(@start_date).and(hdv_t[:InformationDate].lteq(@end_date).and(hdv_t[:CurrentlyFleeing].eq(1)))).
+        where(hdv_t[:InformationDate].gteq(@filter.start).and(hdv_t[:InformationDate].lteq(@filter.end).and(hdv_t[:CurrentlyFleeing].eq(1)))).
         merge(GrdaWarehouse::Hud::Project.viewable_by(current_user)).
         distinct
     end
+
+    private def set_filter
+      @filter = ::Filters::FilterBase.new(
+        filter_params.merge(
+          user_id: current_user.id,
+          default_start: Date.current - 1.month,
+          default_end: Date.current,
+        ),
+      )
+      # Default project type ids [2, 3, 10, 13]
+      @filter.project_type_codes = [:th, :psh, :rrh] unless filter_params[:project_type_codes].present?
```

Comment:
Are these the correct project types to default to for this report?

If you look up at deleted line 14, the previous default was `[2, 3, 13] # TH, PSH, RRH`. This adds `10` into the mix because we consider that part of `psh`.

Reply:
that should be fine.